cabal-install/tests/IntegrationTests2.hs

```diff
@@ -103,7 +103,9 @@ main = do
   removeDirectoryRecursive configDir <|> return ()
   createDirectoryIfMissing True configDir
   -- sigh
-  callProcess "cabal" ["user-config", "init", "-f"]
+  -- NOTE: This is running the `cabal` from the user environment, which is
+  -- generally not the `cabal` being tested!
+  callProcess "cabal" ["-v0", "user-config", "init", "-f"]
```

Comment:
That one was my fault; I'd intended to silence it after verifying it, but forgot. (See https://github.com/haskell/cabal/pull/10205.)

The choice to use the environment `cabal` was deliberate, as I wanted a reliable pristine configuration for the tests and it seemed to me that using the `cabal` under test was incorrect, especially since that is only tested much later.